tracking: {
  local_frame_id: "cepton2",
  global_frame_id: "cepton2",
  # Query Ros TF timeout in ms. ros::Duration time.
  # type: int
  # default: 10
  tf_timeout_ms: 1,

  
  sub_nav_topic: "/navsat/odom",
  sub_nav_queue_size: 1,
  pub_bbox_topic: "/lidar/bboxes",

  # segmented point clouds directly from segment node
  sub_pcs_segmented_topic: "/segment/clouds_segmented",
  sub_pcs_queue_size: 1,
  
  sub_gps_topic: "/Pose",

  # intermediate segments
  pub_segments_coarse_topic: "/tracking/segments_coarse",
  pub_segments_predict_topic: "/tracking/segments_predict",
  pub_segments_topic: "/tracking/segments",
  # output of tracking system
  pub_output_objects_topic: "/tracking/tracking_objects",
  pub_output_trajectories_topic: "/tracking/fixed_trajectories",
  # intermediate visualization
  pub_tracking_objects_topic: "/tracking/objects",
  pub_tracking_objects_cloud_topic: "/tracking/clouds",
  pub_tracking_objects_velocity_topic: "/tracking/objects_vel",
  pub_tracking_objects_tracker_id_topic: "tracking/tracker_ids",
  pub_tracking_objects_trajectory_topic: "/tracking/trajectory",



  pub_lidar_camera_topic_: "/DynamicObj",
  pub_lidar_camera_id_start_: 1,
  pub_lidar_camera_id_num_: 63,
  srv_lidar_camera_name: "/bbox_to_centers",

  care_object_only: true,

  # objects with speeds below this limit will have course = 0
  pub_course_speed_limit: 0.5,

  # lidar's position wrt the cart ground center in meters
  offset: [0.0, 0.0, 0.0],


  
  verbose: false,
  visualize: true,
  use_utm_filter: true,

  utm_left_margin: 0.5,
  utm_right_margin: -0.5,

  

}
